### ISSUE TITLE
[Android] There is a glitch in the homepage footer that affects interaction with footer elements

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/extension-workshop#481]
### Describe the problem and steps to reproduce it:
1. Open Extension Workshop homepage on an Android device 
2. Go down to the footer section
3. Try to scroll up and down while in the footer area and observe its behavior 

### What happened?
The footer is jumpy, some elements might disappear from view.  You might also notice that you can't interact with elements (see the email field in the newsletter section)

### What did you expect to happen?
The homepage footer behaves correctly 

### Anything else we should know?
- I did not reproduce the issue in other EW pages
- I've also checked with Chrome where the jumpiness is even more pronounced (see second screencast)
- reproduced with Fx68 and Chrome, Android 8.0 - all EW servers

[Firefox] - I've tried to capture the part where elements can't be interacted with as expected
![2019-10-02_11-31-00](https://user-images.githubusercontent.com/31961530/66031580-65b15700-e50c-11e9-9b40-86fbade61088.gif)

[Chrome]
![2019-10-02_11-46-09](https://user-images.githubusercontent.com/31961530/66031567-60eca300-e50c-11e9-9fcc-0247a6566aac.gif)

**Comment by AlexandraMoga** [comment]
This is similar to https://github.com/mozilla/extension-workshop/issues/413 but in my issue I've noticed that some elements can't be interacted with, which is a bit more severe.

**Comment by muffinresearch** [comment]
So for jank free scrolling on Android, it seems to work best disabling all of the scroll-based effects.

This will mean things like the menu will be fixed to the top of the viewport. I think this is a reasonable compromise.

**Comment by muffinresearch** [comment]
@AlexandraMoga a fix to just disable the parallax effects will be on -dev in ~12mins or so. When you can, please test it out on -dev and let me know if that improves things enough or if the fixes need to go further.

**Comment by AlexandraMoga** [comment]
@muffinresearch I've checked -dev but I'm still reproducing all the issues described above, unfortunately

**Comment by muffinresearch** [comment]
@AlexandraMoga ok a further reaching solution has landed on -dev. See https://github.com/mozilla/extension-workshop/pull/489 for the details of the approach.

**Comment by AlexandraMoga** [comment]
@muffinresearch here are my results after verifying your patch on -dev:

Improvement: 
- the footer is less jumpy on Firefox

Still an issue: 
- interaction with elements in the 'Connect with us' section is still affected when the footer is scrolled in view
- the jumpiness reported for chromium browsers is still reproducing

I'm understanding that this issue is a bit hard to follow, since it tried to capture too many problems in the same spot. I'm going to open two separate issues for the problems mentioned above with hopefully better STR.
